src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -54,10 +54,24 @@ def generate_config(self, daemon_spec: CephadmDaemonDeploySpec) -> Tuple[Dict[st
                 'value': 'false',
             })
 
+        user, password = '', ''
+        try:
+            user = self.mgr.check_mon_command({
+                'prefix': 'dashboard get-grafana-api-username'
+            }).stdout
+            password = self.mgr.check_mon_command({
+                'prefix': 'dashboard get-grafana-api-password'
+            }).stdout
```

Comment:
Let me check if we can drop the Grafana credentials: they were needed before cephadm, but given the Grafana dashboards are now 'injected' during container build, perhaps we can avoid this.

Reply:
that would be even better!